<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registryPage</class>
 <widget class="QWidget" name="registryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Картотека</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cmbDoubleClickQueueClient">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Новое обращение</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Изменить жалобы/примечания</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Напечатать приглашение</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Перейти в график</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDoubleClickQueueClient">
     <property name="text">
      <string>Двойной щелчок в листе предварительной записи пациента</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkExternalNotificationAuto">
     <property name="text">
      <string>Автоматически открывать вкладку внешних уведомлений</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblOnSingleClientInSearchResult">
     <property name="text">
      <string>После успешного поиска единственного пациента</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkExternalNotificationOnlyAttach">
     <property name="text">
      <string>Показывать только уведомления на прикрепленных пациентов</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="cmbOnSingleClientInSearchResult">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Новое обращение</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Открыть прививочную карту</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblClientsLimit">
     <property name="text">
      <string>Лимит списка пациентов в картотеке</string>
     </property>
     <property name="buddy">
      <cstring>edtClientsLimit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="edtClientsLimit">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100000000</number>
     </property>
     <property name="value">
      <number>10000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbDoubleClickQueueClient</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
